--- qpOASES.pc.in
+++ qpOASES.pc.in
@@ -0,0 +1,11 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=@CMAKE_INSTALL_PREFIX@
+libdir=${prefix}/lib
+includedir=${prefix}/include
+
+Name: @PACKAGE_NAME@
+Description: @PACKAGE_DESCRIPTION@
+Version: @PACKAGE_VERSION@
+Requires: @PKGCONFIG_REQUIRES@
+Libs: -L${libdir} @PKGCONFIG_LIBS@ -lqpOASES
+Cflags: -I${includedir} @PKGCONFIG_CFLAGS@

--- CMakeLists.txt
+++ CMakeLists.txt
@@ -31,6 +31,8 @@
 
 cmake_minimum_required(VERSION 2.6)
 
+add_compile_options(-fPIC)
+
 PROJECT(qpOASES CXX)
 SET(PACKAGE_NAME "qpOASES")
 SET(PACKAGE_VERSION "3.2.0")
@@ -94,7 +96,10 @@
 endif("${isSystemDir}" STREQUAL "-1")
 
 
+CONFIGURE_FILE(${CMAKE_CURRENT_SOURCE_DIR}/${PACKAGE_NAME}.pc.in ${CMAKE_CURRENT_BINARY_DIR}/${PACKAGE_NAME}.pc @ONLY)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${PACKAGE_NAME}.pc DESTINATION lib/pkgconfig)
 
+
 ############################################################
 #################### build and install #####################
 ############################################################

